### PR TITLE
Implement simplemde for blog

### LINKED ISSUE #252: Add Markdown editor
https://simplemde.com/

### PR DESCRIPTION
**Pull request by danielstreif**
**What**:
- Implemented simplemde as a markdown editor to make writing blog entries easier

image

Fixes #252

### CODE REVIEW

**Comment by danielstreif** [comment]
@pheeria Just a first basic version with no extra settings implemented but it works. I had to remove the `required` tag though. Do you know how to implement that for the editor?

**Comment by pheeria** [comment]
> @pheeria Just a first basic version with no extra settings implemented but it works. I had to remove the `required` tag though. Do you know how to implement that for the editor?

I don't know, but I think that's ok 🙂

File: src/views/partials/blogs/create.hbs [hunk 1]
```diff
@@ -1,3 +1,6 @@
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/simplemde/latest/simplemde.min.css">
```
Comment (jarjan) [inline src/views/partials/blogs/create.hbs:1]:
```suggestion
<link rel="stylesheet" href="https://unpkg.com/simplemde@1.11.2/dist/simplemde.min.css">
```

File: src/views/partials/blogs/create.hbs [hunk 2]
```diff
@@ -1,3 +1,6 @@
...
+<script src="https://cdn.jsdelivr.net/simplemde/latest/simplemde.min.js"></script>
```
Comment (jarjan) [inline src/views/partials/blogs/create.hbs:2]:
```suggestion
<link rel="stylesheet" href="https://unpkg.com/simplemde@1.11.2/dist/simplemde.min.css">
<script src="https://unpkg.com/simplemde@1.11.2/dist/simplemde.min.js"></script>
```
I think it is better to use unpkg for all of our static files: less dns lookup = a bit faster website!

Comment (danielstreif) [reply src/views/partials/blogs/create.hbs]: Good call, thanks for the feedback 👍

**Pull request merged**